src/helpers/multiSchemaHelpers.test.ts

```diff
@@ -27,18 +30,55 @@ model Eagle {
 
   @@map("eagles")
   @@schema("birds")
-}`;
+}
+
+model Fish {
+  id   Int    @id
+  name String
+
+  @@map("fish")
+  @@schema("public")
+}
+
+`;
 
 test("returns a list of models with schemas appended to the table name", () => {
   const initialModels = [
     { typeName: "Elephant", tableName: "elephants" },
     { typeName: "Eagle", tableName: "eagles" },
+    { typeName: "Fish", tableName: "fish" },
   ];
 
-  const result = convertToMultiSchemaModels(initialModels, testDataModel);
+  const result = convertToMultiSchemaModels(
+    initialModels,
+    false,
+    "public",
+    null,
+    parseMultiSchemaMap(testDataModel)
+  );
 
   expect(result).toEqual([
     { typeName: "Elephant", tableName: "mammals.elephants" },
     { typeName: "Eagle", tableName: "birds.eagles" },
+    { typeName: "Fish", tableName: "fish" },
+  ]);
+});
+
+test("returns a list of models with schemas appended to the table name filtered by schema", () => {
+  const initialModels = [
+    { typeName: "Elephant", tableName: "elephants" },
+    { typeName: "Eagle", tableName: "eagles" },
+  ];
+
+  const result = convertToMultiSchemaModels(
```

Comment:
nitpick: `convertToMultiSchemaModels` could also accept an options object as it's again hard to quickly scan what the arguments here are
```suggestion
  const result = convertToMultiSchemaModels(
```